src/dev.html

```diff
@@ -12,7 +12,14 @@
 <div id="app"></div>
 <script src="/app.js"></script>
 <script>
-      Elm.Main.init({node: document.getElementById("app"), flags: {services: {"kratia": ""}}});
+    Elm.Main.init({
+        node: document.getElementById("app"), 
+        flags: {
+            services: {
+                kratia: "http://localhost:8080"
```

Comment:
This is good for loal development but will not work if you deploy it. Besides, it needs you to fix CORS headers unlike the proxy setup.

Reply:
Here it should be "", maybe with a comment to change it for local development